### PR TITLE
addional attrs to compliance data

### PR DESCRIPTION
**Pull request by rickmarry**
## Description

Added ipaddress in the form of ipv4 and coded to be equivalent to its corresponding node number, in an effort to store data in a predictable format. 

Other less notable fields were added for completeness sake along with ipaddress 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: lib/compliance_generator.go [hunk 1]
```diff
@@ -18,13 +20,27 @@ const (
...
 	nodeUUID    uuid.UUID
 	platform    string
+	sourceFqdn  string // fqdn of chef server that ran this
+	fqdn        string // fqdn of the box that we're scanning
+	policyName  string
+	policyGroup string
+	orgName     string
+	chefTags    []string
+
+	//// "source_fqdn": "localhost",
```
Comment (alexpop) [inline lib/compliance_generator.go]: Please remove these comments.

File: lib/compliance_generator.go [hunk 2]
```diff
@@ -39,23 +55,115 @@ func GenerateComplianceData(config *Config, requests chan *request) error {
 	return nil
 }
 
+func ip2int(ip net.IP) uint32 {
```
Comment (alexpop) [inline lib/compliance_generator.go]: Is this method still needed?

**Review by alexpop — APPROVED** [review]
Great to have parity of the reports now with the latest coming from Audit cookbook / effortless. Thank you!

File: lib/compliance_generator.go (hunk 1 above)
Comment (rickmarry) [reply lib/compliance_generator.go]: done

File: lib/compliance_generator.go (hunk 2 above)
Comment (rickmarry) [reply lib/compliance_generator.go]: good catch.. removed

**Pull request merged**